vizro-core/src/vizro/models/_components/form/date_picker.py

```diff
@@ -71,33 +72,15 @@ def build(self):
             output=output,
             inputs=inputs,
         )
-        # clientside callback is required as a workaround when the date-picker is overflowing its parent container
-        # if there is not enough space. Caused by another workaround for this issue:
-        # https://github.com/snehilvj/dash-mantine-components/issues/219
-        clientside_callback(
-            ClientsideFunction(namespace="date_picker", function_name="update_date_picker_position"),
-            output=Output(self.id, "dropdownPosition"),
-            inputs=Input(self.id, "n_clicks"),
-        )
```

Comment:
Feel free to completely remove `update_date_picker_position` from `date_picker.js`. 🚀